package.json

```diff
@@ -6,16 +6,16 @@
   "main": "main/index.js",
   "scripts": {
     "preinstall": "npm prune",
-    "postinstall": "electron-rebuild && gulp clean && gulp copy",
-    "test": "jest",
-    "cov": "jest --coverage",
-    "start": "gulp start",
-    "debug": "gulp debug",
-    "package": "gulp package",
-    "package-all": "gulp packageAll",
-    "release": "gulp release",
-    "watch": "gulp watch",
-    "build": "gulp build"
+    "postinstall": "./node_modules/.bin/electron-rebuild && ./node_modules/.bin/gulp clean && ./node_modules/.bin/gulp copy",
```

Comment:
@dstreet `./node_modules/.bin` is redundant 
![image](https://cloud.githubusercontent.com/assets/1622515/13090322/01133e26-d51b-11e5-82bc-ebdb477ef4d5.png)